shap/benchmark/perturbation.py

```diff
@@ -38,16 +39,22 @@ def __init__(self, f, masker, sort_order, score_function, perturbation):
         self.score_values = []
         self.score_aucs = []
         self.labels = []
+
+    def maskedmodel(self):
+        self.masked_model = MaskedModel(self.f, self.masker, shap.links.identity) 
     
-    def score(self, attributions, X, y=None, label=None, silent=False):
+    def score(self, explainer, X, y=None, label=None, silent=False):
+        # if explainer is already the attributions 
```

Comment:
"         # if explainer is already the attributions "
this seems like a naming issue, the explainer should be different from an explanation/attributions.  In which cases is this actually the attributions now, whereas before this was an explainer?

Reply:
This is for the case where users input explainer.values instead of explainer into the input. It is also valid for testing. However, we anticipate users to typically input the original explainer so it is unlikely to happen.